tests/acceptance/helpers/userSettings.js

```diff
@@ -203,28 +213,25 @@ module.exports = {
   /**
    * Gets the password for the un-initialized users and replaces the password accordingly
    *
-   * @param {string} password
+   * @param {string} input
    * @returns {string}
    */
-  replaceInlineCode: function (password) {
-    if (password === '%regular%') {
-      return regularUserPassword
-    }
-    if (password === '%alt1%') {
-      return alt1UserPassword
-    }
-    if (password === '%alt2%') {
-      return alt2UserPassword
-    }
-    if (password === '%alt3') {
-      return alt3UserPassword
-    }
-    if (password === '%alt4%') {
-      return alt4UserPassword
+  replaceInlineCode: function (input) {
+    const codes = {
+      '%regular%': regularUserPassword,
+      '%alt1%': alt1UserPassword,
+      '%alt2%': alt2UserPassword,
+      '%alt3%': alt3UserPassword,
+      '%alt4%': alt4UserPassword,
+      '%alt11%': alt4UserPassword,
+      '%remote_backend_url%': client.globals.remote_backend_url,
+      '%backend_url%': client.globals.backend_url
     }
-    if (password === '%alt11%') {
-      return alt11UserPassword
+    for (const code in codes) {
+      if (input.includes(code)) {
+        input = input.replace(code, codes[code])
+      }
```

Comment:
I would just create a `template` and parse it, and then replace the values here, similar to:
```js
_.templateSettings.interpolate = /%([\s\S]+?)%/g;
var compiled = _.template('hello %user%!');
compiled({ 'user': 'mustache' });

> hello mustache!
```

Another thing, just use object keys without `%<value>%`.

Reply:
@dpakach, better suggestion (`_` is module, but `userSettings` might use `_.template()` in future, so better not to change globals):
```js
const interpolate = /%([\s\S]+?)%/g;
let compiled = _.template(input, { interpolate });
compiled(codes);
```